apps/website/tailwind.config.ts

```diff
@@ -36,9 +36,11 @@ const config: Config = {
           "radial-gradient(50% 50% at 50% 50%, rgba(255, 255, 255, 0.8) 0%, #FFF 100%), linear-gradient(0deg, #FFAF00 0%, #F112D9 32.29%, #4C3FA0 70.83%, #2ED3C6 95.83%)",
         in_touch_background:
           "radial-gradient(50% 50% at 50% 50%, rgba(255, 255, 255, 0.6) 0%, #FFF 100%), linear-gradient(0deg, #4C3FA0 0%, #F112D9 32.29%, #2ED3C6 58.85%, #FFAF00 100%)",
+        error404_background:
```

Comment:
We could probably move these to inline at some point. Non-blocking for this PR though.

Reply:
Yes. 
I think a good moment would be when I'll reverse the prop drilling and make all LP components static like you suggested at some point. 
I kept saving those in the config because I thought they were the same all over that page and it would be easy to reuse them just as a green one and a purple one, but they seem to differ slightly each time I put them in a new component and that pretty much defeats the point of doing it.